README.md

```diff
@@ -176,6 +176,18 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * What are the pros and cons of using Promises instead of callbacks?
 * What are some of the advantages/disadvantages of writing JavaScript code in a language that compiles to JavaScript?
 * What tools and techniques do you use debugging JavaScript code?
+* What are falsy values
+
+Falsy values are values that evaluate to false (using == or in if()):
```

Comment:
when converted to boolean type at the time of e.g. conversion by double exclamation mark or in if statement